game_event/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+from court.models import Court
+from player_rating.models import Rating
+
+
+class BallGame(models.TextChoices):
```

Comment:
Isn't it already defined in the Player model?
you can just import it

Reply:
@MaayanMashhadi @amitKaplansky I merged before seeing this comment. Please check it and open an issue accordingly.